ts/features/pushNotifications/analytics/index.ts

```diff
@@ -90,3 +92,73 @@ export const trackNotificationPermissionsStatus = (
   });
   void mixpanelTrack(eventName, props);
 };
+
+export const trackPushNotificationsBannerVisualized = (
+  bannerPage: typeof MESSAGES_ROUTES.MESSAGES_HOME | typeof ROUTES.SETTINGS_MAIN
+) => {
+  const eventName = "BANNER";
+  const props = buildEventProperties("UX", "screen_view", {
+    banner_id: "push_notif_activation",
+    banner_page: bannerPage,
+    banner_landing: "os_notification_settings"
+  });
+  void mixpanelTrack(eventName, props);
+};
+
+export const trackPushNotificationsBannerTap = (
+  bannerPage: typeof MESSAGES_ROUTES.MESSAGES_HOME | typeof ROUTES.SETTINGS_MAIN
+) => {
+  const eventName = "TAP_BANNER";
+  const props = buildEventProperties("UX", "action", {
+    banner_id: "push_notif_activation",
+    banner_page: bannerPage,
+    banner_landing: "os_notification_settings"
+  });
+  void mixpanelTrack(eventName, props);
+};
+
+export const trackPushNotificationsBannerClosure = () => {
+  const eventName = "CLOSE_BANNER";
+  const props = buildEventProperties("UX", "action", {
+    banner_id: "push_notif_activation",
+    banner_page: MESSAGES_ROUTES.MESSAGES_HOME,
```

Comment:
since this banner is not attached to the`messages_home` screen, but rather to the current landing screen, would it make more sense to add the current route as a parameter?

Reply:
The event is related to the user manually closing the banner but (the banner) appears only in the messages' home and in the settings screen. Latter does not offer an option to close the banner so we do not have a use-case where the banner can be close anywhere but the messages' home

Reply:
Done on 4478970